includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2107,6 +2119,30 @@ protected function get_account_branding_secondary_color( $default_value = '' ):
 		return $default_value;
 	}
 
+	/**
+	 * Retrieves the domestic currency of the current account based on its country.
+	 * It will fallback to the store's currency if the account's country is not supported.
+	 *
+	 * @return string The domestic currency code.
+	 */
+	protected function get_account_domestic_currency(): string {
+		$merchant_country    = $this->account->get_account_country();
+		$country_locale_data = $this->localization_service->get_country_locale_data( $merchant_country );
+
+		// Check for missing country locale data.
+		if ( ! $country_locale_data || ! isset( $country_locale_data['currency_code'] ) ) {
```

Comment:
Can be simplified as `if ( ! isset( $country_locale_data['currency_code'] ) ) {`

Reply:
Done here https://github.com/Automattic/woocommerce-payments/pull/6920/commits/0318cfaab665c4b57586954c375059d0006d0fde thanks!